src/renderer/components/query.jsx

```diff
@@ -157,6 +157,34 @@ export default class Query extends Component {
     });
   }
 
+  getCommands () {
```

Comment:
for sure, it would also impact, if you press the command without the focus in the ace editor, it would need some actions to change some store state through reducers to enable increase the font...

if it worth the effort i can implement... but i'm not sure if it worth